### ISSUE TITLE
[Feature Request] [VTabsSlider] Add prop to override default tabs-slider-height

### ISSUE THREAD

**Issue by JeremyWalters** [vuetifyjs/vuetify#7683]
### Problem to solve
The ability to adjust VTabsSlider `height/thickness`

### Proposed solution
Add prop to VTabsSlider that would override `tabs-slider-height`.

**Comment by jacekkarczmarczyk** [comment]
There's a `$tabs-slider-height` SASS variable available in 2.0, not sure if want a prop for this

**Comment by MatthewAry** [comment]
@jacekkarczmarczyk Where can a master list of variables like these be found?

**Comment by jacekkarczmarczyk** [comment]
Some variables are defined in `src/styles/settings/*.scss` and some in component's folder, for example `src/components/VTabs/_variables.scss`

**Comment by MatthewAry** [comment]
I've been unsuccessful in overriding the sass variables after following the directions listed here. https://vuetifyjs.com/en/customization/sass-variables#setup-vue-config-js I'm getting errors about missing semicolons in the style files in the vuetify package.

Is this because of #7795 and https://github.com/vuejs/vue-cli/issues/4116 ?

**Comment by MatthewAry** [comment]
So I've taken steps to change the variable as suggested.  See below.

```
// src/sass/main.scss
@import '~vuetify/src/styles/styles.sass';

$tabs-slider-height: 3px;
```

Yet, doing this does not adjust the tab slider height. I don't think this variable is actually used because the HTML that is generated for tab sliders looks something like this.

```
<div class="v-tabs-slider-wrapper" style="height: 2px; left: 885px; width: 214px;"><div data-v-0e4e1fa4="" class="v-tabs-slider primary"></div></div>
```
Which suggests that the slider height will get clipped anyways even if the style did take effect.

When I used the variable I confirmed that it did make a change but it could not take effect.

![Screen Shot 2019-07-25 at 7 24 58 AM](https://user-images.githubusercontent.com/157217/61878844-09e4a500-aeaf-11e9-8c7c-cc0f8eb66593.png)
